### PR TITLE
Preface changes

### PR DESCRIPTION
**Pull request by mhoeber**
@lamagnifica

### CODE REVIEW

**Comment by lamagnifica** [comment]
please create a companion PR to make the same changes to the Preface in the edx-analytics-dashboard repo

**Comment by mhoeber** [comment]
@lamagnifica   ?? that's what this is

bwahaha -- well then create a companion PR to make whatever changes here in the edx-documentation repo! --AH

File: docs/en_us/shared/preface.rst [hunk 1]
```diff
@@ -13,14 +13,83 @@ goals, and solve problems, edX provides a variety of information resources.
...
 resources described in this preface.
 
+.. _The edX Partner Portal:
+
+***********************
+The edX Partner Portal
+***********************
+
+The `edX Partner Portal`_ is the destination for partners to learn, connect,
+and collaborate with one another. Partners can explore rich resources and share
```
Comment (lamagnifica) [inline docs/en_us/shared/preface.rst]: possibly, "Partners ^of edX^..." or "Representatives at edX partner institutions..."? "partner" alone isn't defined (actually, partner could be a glossary term)
In the Research guide, I use phrases like "A data czar is the single representative at a partner institution who ..." and "This document is intended for researchers, data czars, and administrative teams at edX partner institutions who use the edX data exports to gain insight into their courses and students."

File: docs/en_us/shared/preface.rst [hunk 2]
```diff
@@ -13,14 +13,83 @@ goals, and solve problems, edX provides a variety of information resources.
...
+success stories and best practices while staying up-to-date with important news
+and updates.
+
+To use the edX Partner Portal, you must register and request verification as an
+edX partner. If you are an edX partner and have not used the edX Partner
+Portal, follow these steps.
```
Comment (lamagnifica) [inline docs/en_us/shared/preface.rst]: ...Partner Portal ^yet^, ... (?)

**Comment by mhoeber** [comment]
@lamagnifica   updated those lines

File: docs/en_us/shared/preface.rst [hunk 3]
```diff
@@ -13,14 +13,83 @@ goals, and solve problems, edX provides a variety of information resources.
...
+   your account access within 24 hours.
+
+.. _The Open edX Portal:
+
+***********************
+The Open edX Portal
+***********************
+
+The `Open edX Portal`_ is the destination for all edX users to learn about the
+edX roadmap, as well as hosting, extending the edX platform, and contributing
```
Comment (lamagnifica) [inline docs/en_us/shared/preface.rst:54]: this reads a bit awkwardly. maybe, "The `Open edX Portal`_ is the destination for all edX users to review the edX roadmap and learn about hosting, extending the edX platform, and contributing...

File: docs/en_us/shared/preface.rst [hunk 4]
```diff
@@ -13,14 +13,83 @@ goals, and solve problems, edX provides a variety of information resources.
...
+#. Select **Create New Account**. You will receive a confirmation email with
+   your account access within 24 hours.
+
+.. _The Open edX Portal:
+
+***********************
+The Open edX Portal
+***********************
+
+The `Open edX Portal`_ is the destination for all edX users to learn about the
```
Comment (lamagnifica) [inline docs/en_us/shared/preface.rst:53]: what about a different word than "users"? community members?
... the destination for all members of the edX community to...

File: docs/en_us/shared/preface.rst [hunk 5]
```diff
@@ -13,14 +13,83 @@ goals, and solve problems, edX provides a variety of information resources.
...
+edX roadmap, as well as hosting, extending the edX platform, and contributing
+to Open edX. In addition, the Open edX Portal provides product announcements,
+the Open edX blog, and other rich community resources.
+
+All users can view content on the Open edX Portal without creating an account
```
Comment (lamagnifica) [inline docs/en_us/shared/preface.rst:58]: All members of the edX community...

File: docs/en_us/shared/preface.rst [hunk 6]
```diff
@@ -13,14 +13,83 @@ goals, and solve problems, edX provides a variety of information resources.
...
+
+To comment on blog posts or the edX roadmap, you must create an account and log
+in. If you do not have an account, follow these steps.
+
+#. Visit `open.edx.org/user/register`_.
+#. Fill in your personal details.
+#. Select **Create New Account**. You are then logged in to the `Open edX
+   Portal`_.
+
+***************************************************
```
Comment (lamagnifica) [inline docs/en_us/shared/preface.rst:69]: missing the xref link above?

File: docs/en_us/shared/preface.rst [hunk 7]
```diff
@@ -13,14 +13,83 @@ goals, and solve problems, edX provides a variety of information resources.
...
+Release Announcements through the Open edX Portal
+***************************************************
+
+To receive and share product and release announcements by email, subscribe to
```
Comment (lamagnifica) [inline docs/en_us/shared/preface.rst:73]: ...email, ^you can^ subscribe...

File: docs/en_us/shared/preface.rst [hunk 8]
```diff
@@ -13,14 +13,83 @@ goals, and solve problems, edX provides a variety of information resources.
...
+announcements on the `Open edX Portal`_.
+
+#. Create an account on the `Open edX Portal`_ as described above.
+#. Go to https://open.edx.org/announcements.
```
Comment (lamagnifica) [inline docs/en_us/shared/preface.rst:77]: link the URL?

File: docs/en_us/shared/preface.rst [hunk 9]
```diff
@@ -13,14 +13,83 @@ goals, and solve problems, edX provides a variety of information resources.
...
+#. Under **Announcement Type** in the **Subscriptions** block, select the type
```
Comment (lamagnifica) [inline docs/en_us/shared/preface.rst:78]: plural "types"

File: docs/en_us/shared/preface.rst [hunk 10]
```diff
@@ -13,14 +13,83 @@ goals, and solve problems, edX provides a variety of information resources.
...
+   of announcements that you want to receive through email.
+
+   .. image:: ../../../shared/images/Announcement_subscriptions.png
+     :alt: Subscription block in the Open edX Portal Announcements page.
+
+4. Click **Save**.
+   
+You will now receive email messages when new announcements of the types you
```
Comment (lamagnifica) [inline docs/en_us/shared/preface.rst:86]: make less passive?
You will automatically receive an email message whenever edX adds announcements of the types you select.

File: docs/en_us/shared/preface.rst [hunk 11]
```diff
@@ -93,11 +162,8 @@ To receive and share information by email, course team members can:
 * Join the `openedx-studio`_ Google group to ask questions and participate in
   discussions with peers at other edX partner organizations and edX staffers.
 
-Wikis and Web Sites
--------------------
-
-The edX product team maintains the `Open edX Product`_ wiki, which includes the
-`Open edX Public Product Roadmap`_.
+Course Author Support
+----------------------
```
Comment (lamagnifica) [inline docs/en_us/shared/preface.rst:167]: on line 202 below, remove the reference to the roadmap?

File: docs/en_us/shared/preface.rst [hunk 12]
```diff
@@ -201,29 +262,49 @@ staffers.
...
-
-The edX Engineering team maintains the `Open Source Home`_ wiki, which provides
-insights into the plans, projects, and questions that the edX Open Source team
-is working on with the community.
-
 The pull request dashboard_  is a visualization of the count and age of the
 pull requests (PRs) assigned to teams at edX. Click the bars in this chart to
 get more information about the PRs.
 
 .. _Resources for Students:
```
Comment (lamagnifica) [inline docs/en_us/shared/preface.rst:283]: move this ^^ xref link to line 307-ish

File: docs/en_us/shared/preface.rst [hunk 13]
```diff
@@ -201,29 +262,49 @@ staffers.
...
 
+.. _Resources for Open edX:
+
+**************************
+Resources for Open edX
+**************************
+
+Hosting providers, platform exenders, core contributors, and course staff all
```
Comment (lamagnifica) [inline docs/en_us/shared/preface.rst:291]: exenders --> extenders

File: docs/en_us/shared/preface.rst [hunk 14]
```diff
@@ -13,14 +13,83 @@ goals, and solve problems, edX provides a variety of information resources.
...
+To receive and share product and release announcements by email, subscribe to
+announcements on the `Open edX Portal`_.
+
+#. Create an account on the `Open edX Portal`_ as described above.
+#. Go to https://open.edx.org/announcements.
+#. Under **Announcement Type** in the **Subscriptions** block, select the type
+   of announcements that you want to receive through email.
+
+   .. image:: ../../../shared/images/Announcement_subscriptions.png
+     :alt: Subscription block in the Open edX Portal Announcements page.
```
Comment (lamagnifica) [inline docs/en_us/shared/preface.rst:82]: personally, I don't think the image is necessary. It is somewhat likely to change, too, as new categories are added, or we rename or conflate some of these...

**Comment by lamagnifica** [comment]
One spelling error to fix, then a bunch of suggestions. :+1:

**Comment by lamagnifica** [comment]
@mhoeber can this be merged?

**Comment by dsjen** [comment]
I just noticed that this hasn't been merged.  Is the intention to merge it?

**Comment by lamagnifica** [comment]
@mhoeber is this still valid? you recently created a new PR in this repo for the preface -- might all of these changes have been included at that time?

**Comment by mhoeber** [comment]
oh wow this is old.  didn't realize it was still there.  latest update was complete copy from doc repo.  So I will close this

**Pull request closed without merging**